modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -860,18 +823,109 @@ class CV_EXPORTS_W QRCodeDetector
             OutputArray points = noArray(),
             OutputArrayOfArrays straight_qrcode = noArray()
     ) const;
-
-protected:
     struct Impl;
+protected:
     Ptr<Impl> p;
 };
 
+class CV_EXPORTS_W QRCodeDetector : public QRCodeDetectorBase
+{
+public:
+    CV_WRAP QRCodeDetector();
+    ~QRCodeDetector();
+
+
+    /** @brief sets the epsilon used during the horizontal scan of QR code stop marker detection.
+     @param epsX Epsilon neighborhood, which allows you to determine the horizontal pattern
+     of the scheme 1:1:3:1:1 according to QR code standard.
+    */
+    CV_WRAP void setEpsX(double epsX);
```

Comment:
All "set" methods should return `*this` reference to allow construction chains.

Reply:
these setters drop CI
Maybe update the setters later?

Reply:
In DNN `pyopencv_cv_dnn_dnn_ClassificationModel_setEnableSoftmaxPostProcessing` has been properly generated.

Reply:
done.
The directive `CV_EXPORTS_W_SIMPLE` instead of `CV_EXPORTS_W` directive helped to add setter